python/frame_pipeline_evolution.py

```diff
@@ -4,42 +4,99 @@
 
 set_log_level(LogLevel.Trace)
 
-from savant_rs.pipeline import VideoPipelineStagePayloadType, VideoPipeline, VideoPipelineConfiguration, StageFunction
-from savant_rs.primitives import VideoFrame, VideoFrameContent, VideoFrameTranscodingMethod, VideoFrameTransformation, \
-    Attribute, AttributeValue
+from savant_rs.pipeline import (
+    VideoPipelineStagePayloadType,
+    VideoPipeline,
+    VideoPipelineConfiguration,
+    StageFunction,
+)
+from savant_rs.primitives import (
+    VideoFrame,
+    VideoFrameContent,
+    VideoFrameTranscodingMethod,
+    VideoFrameTransformation,
+    Attribute,
+    AttributeValue,
+)
 from savant_rs.utils import gen_frame, TelemetrySpan, enable_dl_detection
-from savant_rs import init_jaeger_tracer
+
+# from savant_rs import init_jaeger_tracer
 
 if __name__ == "__main__":
     savant_rs.version()
     enable_dl_detection()  # enables internal DL detection (checks every 5 secs)
-    log(LogLevel.Info, "root", "Begin operation", dict(savant_rs_version=savant_rs.version()))
-    init_jaeger_tracer("demo-pipeline", "localhost:6831")
+    log(
+        LogLevel.Info,
+        "root",
+        "Begin operation",
+        dict(savant_rs_version=savant_rs.version()),
+    )
+    # init_jaeger_tracer("demo-pipeline", "localhost:6831")
```

Comment:
I suggest to remove it at all (together with import) or make the code (even commented) correct.